### PR TITLE
Prepare to release 0.11.0

### PR DESCRIPTION
**Pull request by sjhx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 # v-next
 ##
 
+# v0.11.1
+## 2021-06-14
+* Further documentation improvements including godoc
+* Update webhook resources to v1 
+* update dependencies and glonag version to 1.16.5
```
Comment (aaroncollins) [inline CHANGELOG.md:21]:
```suggestion
* update dependencies and golang version to 1.16.5
```

**Review by aaroncollins — CHANGES_REQUESTED** [review]
All looks good otherwise

**Review by aaroncollins — DISMISSED** [review]

**Review by jhart1685 — CHANGES_REQUESTED** [review]
Why not 0.11.0 ?

**Review by jhart1685 — APPROVED** [review]
lgtm

**Pull request merged**